roles/nginx/tasks/main.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: Add NGINX PPA
+  apt_repository:
+    repo: "ppa:nginx/stable"
+
+- name: Install NGINX
+  package:
+    name: nginx
+    state: latest
```

Comment:
Using 'latest` is generally a bad practice in ansible (though I understand passable in this case based on the ppa). Refer to #6.